fem/estimators.hpp

```diff
@@ -426,13 +425,13 @@ class LpErrorEstimator : public ErrorEstimator
         ||∇(u-uₕ)||ₑ ≦ √( C hₑ ∑ₖ (hₖ ∫ |J[∇uₕ]|²) dS )
     where "e" denotes an element, ||⋅||ₑ the corresponding local norm and k the
     corresponding faces. u is the analytic solution and uₕ the discretized
-    solution. hₖ and hₑ are factors dependend on the face and element geometry.
+    solution. hₖ and hₑ are factors dependent on the face and element geometry.
     J is the jump function, i.e. the difference between the limits at each point
     for each side of the face. A custom method to compute hₖ can be provided. It
     is also possible to estimate the error only on a subspace by feeding this
     class an attribute array describing the subspace.
 
-    @note This algorithm is only for Poisson problems a proper error esimator.
+    @note This algorithm is a proper error esimator only for Poisson problems.
```

Comment:
What is meant by "proper" here? I think it that you can only get the upper bound with lowest order FEM for Poisson.